checkout/checkin.md

```diff
@@ -31,6 +31,9 @@ The payer will be identified with the `consumers` resource and will be
 persisted to streamline future Payment Menu processes. Payer identification
 is done through the `initiate-consumer-session` operation.
 
+If the end user is using the Payment Menu as a guest, you can go directly to
+step 3, which you will find on the next page.
```

Comment:
Turn this into an informational alert so it's more visible, perhaps?

Reply:
Done!